server/driver/configuration.cpp

```diff
@@ -138,6 +138,11 @@ configuration configuration::read_user_configuration()
 		else if (auto it = json.find("tcp_only"); it != json.end())
 			result.tcp_only = *it;
 
+		if (auto it = json.find("force-palm"); it != json.end())
+			result.force_palm = *it;
+		else if (auto it = json.find("force_palm"); it != json.end())
+			result.force_palm = *it;
```

Comment:
We don't need the version with underscores, `tcp_only` has it for compatibility, but from now on I would prefer to only have dashes in the json.

I am wondering if we should pass the rotation in the configuration and generate the palm only if configured